quickstarts/spring-with-s2i.yaml

```diff
@@ -0,0 +1,208 @@
+  introduction: >-
+    **Spring** is a Java framework for building applications based on a distributed microservices architecture. 
+    
+    - Spring enables easy packaging and configuration of Spring applications into a self-contained executable application which can be easily deployed as a container to OpenShift.
+    
+    - Spring applications can integrate OpenShift capabilities to provide a natural "Spring on OpenShift" developer experience for both existing and net-new Spring applications. For example:
+    
+    - Externalized Configuration using Kubernetes ConfigMaps and integration with Spring Cloud Kubernetes
```

Comment:
This intro seems like a wealth of information, but how much of it is entirely relevant to this quick start? Can we unify the bulleted list by starting with the same article of speech and really zeroing in on how getting started with Spring is valuable to OpenShift users?